resources/js/src/app/components/common/GoogleMaps.js

```diff
@@ -0,0 +1,160 @@
+Vue.component("google-maps-widget",
+    {
+        template: `<div :class="aspectRatio" class="maps-component" ref="googleMapsContainer"></div>`,
+
+        props:
+        {
+            googleApiKey:
+                {
+                    type: String,
+                    required: true
+                },
+            address:
+                {
+                    type: String,
+                    required: false
+                },
+            lat:
+                {
+                    type: Number
+                },
+            lng:
+                {
+                    type: Number
+                },
+            zoom:
+                {
+                    type: Number,
+                    default: 16
+                },
+            aspectRatio:
```

Comment:
Sollte in Widget-Config ausgelagert werden. Siehe ImageBoxWidget

Reply:
done